clang/test/CodeGen/RISCV/riscv-vector-callingconv.cpp

```diff
@@ -0,0 +1,19 @@
+// REQUIRES: riscv-registered-target
```

Comment:
Does this test really require `riscv-registered-target`? It doesn't include any headers and doesn't run any IR passes